docs/index.rst

```diff
@@ -51,7 +51,8 @@ Installation
 Requirements:
 
 * Python 2.7+ or 3.3+
-* `Django Rest Framework`_
+* `Django`_ 1.8 (NB: hawkrest is not yet compatible with Django 1.9)
+* `Django Rest Framework`_ 3.2 or 3.3
```

Comment:
it's worth noting something like "older versions of these libraries may work but support is not guaranteed."